### DISCUSSION TITLE
Chrome Web Store warning.

### DISCUSSION THREAD

**Discussion by flatheadmill** [glacambre/firenvim#1641] [General]
Chrome Web Store is saying...

> This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions.

I don't see any discussion of this. I'm using Chrome on Mac and Linux. Are people mostly using Firefox?

**Comment by glacambre** [comment]
Hi, thanks for the report. I indeed only use Firefox :). This warning is caused by Firenvim being a Manifest v2 extension, fixing it requires porting Firenvim to manifest v3 (tracked under https://github.com/glacambre/firenvim/issues/1193 ). I've delayed this as much as possible because I'm lazy and Firenvim will likely lose some features with the port, but I really should start working on that...
